src/js/utils/index.ts

```diff
@@ -0,0 +1,29 @@
+import { Utils } from "@mat3ra/utils";
+
+import { TestCaseHandler } from "./TestCaseHandler";
+
+export function generateTestFeaturesFromYAMLConfig(yamlContent: string, templateContent: any) {
```

Comment:
Let's update this to use JS objects instead of YAML.

This function should accept the following arguments:

- templateContent: object
- testCaseSchema: object
- testCaseConfigs: array